data_summaries/OpenMathInstruct-1.json

```diff
@@ -0,0 +1,125 @@
+{
+    "omi-1-math": {
+        "Unique Dataset Identifier": "omi-1-math",
+        "Dataset Name": "omi-1-math",
+        "Paper Title": "OpenMathInstruct-1: A 1.8 Million Math Instruction Tuning Dataset",
+        "Dataset URL": "https://huggingface.co/datasets/nvidia/OpenMathInstruct-1/viewer/default/train?f[dataset][value]=%27math%27",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/nvidia/OpenMathInstruct-1",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2402.10176",
+        "Semantic Scholar Corpus ID": 267681752,
+        "Collection": "OpenMathInstruct-1",
+        "Collection URL": "https://huggingface.co/datasets/nvidia/OpenMathInstruct-1",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Algebraic Expression Evaluation",
+            "Instruction Following",
+            "Logical and Mathematical Reasoning",
+            "Mathematical Equation Generation",
+            "Probabilistic Reasoning",
+            "Question Answering"    
+        ],
+        "Text Sources": [
+            "artofproblemsolving.com/community/c3158_usa_contests",
+            "american invitational mathematics examination (AIME)",
+            "american mathematics competitions (AMC)",
+            "mathematics competitions",
+            "math"
```

Comment:
Is this a website/book/text source? Or crowdsourced math questions? We probably don't need to add just "math", but instead the source or "crowdsourced"

Reply:
Oh I see it's a dataset. Fine to just have the derived from "math" dataset in that part only

Reply:
Yes, the math in the text sources refers to the "MATH" dataset, and AIME and AMC are where MATH got the problems. Should I keep math only as a text source?